fitbook1/src/util/findBookMarked.ts

```diff
@@ -0,0 +1,11 @@
+const findBookMarked = (postId: string, bookmarks: []) => {
+  // TODO change any
+
+  const foundElement = bookmarks.find((item: any) => {
```

Comment:
here item's type can be replaced with an object

Reply:
I will do it later, currently i am have to complete p0 features first for roc8 onboarding